### DISCUSSION TITLE
Order of custom and Directus migrations

### DISCUSSION THREAD

**Discussion by rgieseke** [directus/directus#5675] [Feature Requests]
Currently custom migrations are run after Directus' migrations.
https://github.com/directus/directus/blob/801e86855452bd32d49475d8b07ce151c11e962a/api/src/database/migrations/run.ts#L27

Maybe they could together be sorted alphanumerically?

So one could have 

```
20210416A-add-collections-accountability.ts
20210422A-remove-files-interface.ts
20210422A01-my-custom-migration.js
20210422A02-another-custom-migration.js
20210506A-rename-interfaces.ts
```
Then one could develop the individual database migrations including changes to Directus fields or relation settings in sync with changes of Directus' database structure.

Using `20210422A01` as an example for a custom migration to be run on the same day, not knowing yet whether there might be  `20210422B-directus-migration`.

**Comment by rijkvanzanten** [comment]
Makes total sense to me! Would you like to open a PR for that @rgieseke?

**Reply by rgieseke** [reply]
Sure, thanks for the quick feedback!

**Reply by rgieseke** [reply]
... and thanks for the fast turnaround in https://github.com/directus/directus/pull/5685 :smiley:

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
